neo/IO/Helper.cs

```diff
@@ -21,6 +21,16 @@ public static T AsSerializable<T>(this byte[] value, int start = 0) where T : IS
             }
         }
 
+        public static unsafe T AsSerializable<T>(this ReadOnlySpan<byte> value) where T : ISerializable, new()
+        {
```

Comment:
@erikzhang According to my test, if the `ReadOnlySpan` is empty. it throw an exception on `new UnmanagedMemoryStream(pointer, value.Length))`

Reply:
Fixed